packages/client/src/schedule-client.ts

```diff
@@ -497,6 +497,7 @@ export class ScheduleClient extends BaseClient {
   }
 
   protected rethrowGrpcError(err: unknown, scheduleId: string, fallbackMessage: string): never {
+    if (err instanceof TypeError) throw err;
```

Comment:
Hmm.. wondering if we can catch the type error at a different level.

Reply:
We could, but then we would need that extra check in every function that does encodings. Not a big thing, but that's why I put it there.

Reply:
Just find it a bit out of place here.
I’ll let you make the call.